backend/src/Squidex/Program.cs

```diff
@@ -61,6 +63,16 @@ public static void Main(string[] args)
                 })
                 .ConfigureWebHostDefaults(builder =>
                 {
+                    builder.ConfigureKestrel((context, serverOptions) =>
+                    {
+                        if (context.Configuration.GetValue<bool>("devMode:enable"))
```

Comment:
Why do we need this? Shouldn't be there the development mode already?

Reply:
this is required so that self signed certificate is only served in development mode.

Reply:
we can probably use ASPNETCORE_ENVIRONMENT variable to check if it's running in dev mode and then server certificate.

Reply:
Yes, but I mean there is already a development in ASP.NET Core, e.g.

https://github.com/Squidex/squidex/blob/835274396f58f9b9f8f11bab8956639337a97a5b/backend/src/Squidex/Areas/Frontend/Startup.cs#L41

We can introduce an additional flag, but at least we should check for both.

if (context.HostingEnvironment.isDevelopment() || ...)